Copy old docs (#7)
pgruetter/sdl-docs: docs/reference/deploy-microsoft-azure.md

---
id: deploy-microsoft-azure

> title: Deploy on Microsoft Azure Databricks

Reviewer: This chapter seems a bit outdated

Author: Yes, absolutely. Let's merge it as it is and all of the "deactivated" pages will be reviewed or rewritten before they're made visible in the sidebar.

---

Smart Data Lake can be executed on the Databricks Service running on Microsoft Azure.
The following steps show how to execute a simple copy feed.

At the time of this writing, a few extra steps are needed to overwrite specific libraries.
When running a job in Databricks, a few dependencies are given and can not be simply overwritten with your own as described in the
[Azure documentation](https://docs.microsoft.com/en-us/azure/databricks/jobs#library-dependencies).
Since we use a newer version of typesafe config, we need to force the overwrite of this dependency.
We will create a cluster init script that downloads the library and saves it on the cluster, then use Sparks ChildFirstURLClassLoader to explicitly load our library first.
This can hopefully be simplified in the future.

1. In your Azure portal, create a Databricks Workspace and launch it

> 2. Create a cluster as you see fit, for a first test you can use the miminal configuration of 1 Worker and 1 Driver node.

Reviewer: ```suggestion
2. Create a cluster that fits your needs. For a first test you can use the miminal configuration of 1 Worker and 1 Driver node.
```
